php/php-rr-sdk.md

```diff
@@ -466,13 +446,13 @@ the explicit yield required.
 ```php
 public function run(string $input): string
 {
-    $session = new Session([
-        Session::START_TO_FINISH_TIMEOUT => 300,
+    $session = Workflow::newSession([
+        Session::START_TO_FINISH => 300,
```

Comment:
I would use the same names as the already existing Go impl: 	SessionOptions struct {
		ExecutionTimeout time.Duration
		CreationTimeout  time.Duration
		HeartbeatTimeout time.Duration
	}

Reply:
These names are not consistent with Java implementation.